### PR TITLE
Add litepod config for 70B model

### PR DESCRIPTION
**Pull request by kelvin-zou**
Adding a litepod config to avoid remat on GPU or v5p, also fixing a nit issue on flashattention remat_spec.

### CODE REVIEW

File: axlearn/experiments/text/gpt/fuji.py [hunk 1]
```diff
@@ -403,4 +401,24 @@ def make_grad_accum_config(
...
             )
 
+        if model_size == "70B":
+
+            def make_config_with_act_offload(base_config_name: str) -> SpmdTrainer.Config:
+                """Make configs for the v5e/v6e tpu with low HBM."""
+                # pytype: disable=annotation-type-mismatch
+                cfg: SpmdTrainer.Config = config_map[base_config_name]().clone()
+                # pytype: enable=annotation-type-mismatch
+                update_model_remat_config(
```
Comment (samos123) [inline axlearn/experiments/text/gpt/fuji.py:411]: Would it call update_model_remat_config twice this way?

Comment (kelvin-zou) [reply axlearn/experiments/text/gpt/fuji.py:411]: unfortunately yes, but those are light weight config changes.

**Comment by kelvin-zou** [comment]
@samos123 Yes, I am working on some more general way to support various HW specific configs, will share it once I have it approved internally.

File: axlearn/experiments/text/gpt/fuji.py (hunk 1 above)
Comment (samos123) [reply axlearn/experiments/text/gpt/fuji.py:411]: hmm as long as it doesn't cause issues, it may be fine?

**Comment by kelvin-zou** [comment]
Close it for a more thorough PR later. No need for it after the other change.

**Pull request closed without merging**